### PR TITLE
porting master into v3

### PR DESCRIPTION
**Pull request by panquez**

### CODE REVIEW

File: docs/how_to_create_module.dox [hunk 1]
```diff
@@ -73,7 +75,6 @@ Template repository contains some files. Here is information on these files:
 
  - `LICENSE` - In this file you specify the Licence of your module 
  - `README.md` - This file is used as overview of your GitHub project. Add any information you consider necessary.
- - `azure-pipelines.yml` - Configuration file for running jobs in Azure DevOps pipelines (for formatting, testing and much more).
```
Comment (BotellaA) [inline docs/how_to_create_module.dox]: Add something about .github/workflows/{CI.yml,CD.yml}?

Comment (panquez) [reply docs/how_to_create_module.dox]: Done

File: docs/how_to_create_module.dox [hunk 2]
```diff
@@ -73,10 +75,10 @@ Template repository contains some files. Here is information on these files:
...
  - `.clang-format` - Format rules file used by clang. Run it by your own or automatize formatting using this file and CI tools.
  - `.gitattributes` - List of attributes on files and folders (e.g. ignored by GitHub for repository languages) 
  - `.gitignore` - List of files and folders ignored by Git.
  - `.releaserc` - Configuration file for semantic-release. Have a look at https://github.com/semantic-release/semantic-release it's great!
+ - `.github/workflows/{CI.yml,CD.yml}` - Configuration files for Github Actions (https://github.com/features/actions) for core formatting, core testing, release generation and upload, ... Be aware that the variables used in these files are configured for `OpenGeode-ModuleTemplate` repository. They have to be modified depending on your project and your needs.
```
Comment (BotellaA) [inline docs/how_to_create_module.dox]:
```suggestion
 - `.github/workflows/{CI.yml,CD.yml}` - Configuration files for Github Actions (https://github.com/features/actions) for code formatting, code testing, release generation and upload, ... Be aware that the variables used in these files are configured for `OpenGeode-ModuleTemplate` repository. They have to be modified depending on your project and your needs.
```

**Pull request merged**